.env.example

```diff
@@ -22,6 +22,9 @@ MS_GRAPH_CLIENT_SECRET=
 ZOOM_CLIENT_ID=
 ZOOM_CLIENT_SECRET=
 
+#Used for the Daily integration
+DAILY_API_KEY=
```

Comment:
This is awesome! @lunchpaillola 

Would you kindly add instructions on how to setup the integration like the other ones in the `README.md`?

Reply:
Definitely, looks like I accidentally removed that from some previous commits but it's up now!